src/main/resources/mixins.tweakeroo.json

```diff
@@ -1,82 +1,83 @@
 {
-	"required": true,
-	"package": "fi.dy.masa.tweakeroo.mixin",
-	"minVersion": "0.8",
-	"compatibilityLevel": "JAVA_17",
-	"client": [
-		"IMixinAbstractBlock",
-		"IMixinAxeItem",
-		"IMixinChunkLightProvider",
-		"IMixinClientWorld",
-		"IMixinCommandBlockExecutor",
-		"IMixinAbstractHorseEntity",
-		"IMixinShovelItem",
-		"IMixinSimpleOption",
-		"MixinAbstractClientPlayerEntity",
-		"MixinAbstractInventoryScreen",
-		"MixinAbstractSignEditScreen",
-		"MixinBackgroundRenderer",
-		"MixinBatEntity",
-		"MixinBeaconBlockEntityRenderer",
-		"MixinBlockEntityRenderDispatcher",
-		"MixinBlockItem",
-		"MixinBossBarHud",
-		"MixinBuiltChunk",
-		"MixinChatHud",
-		"MixinChatScreen",
-		"MixinChunkBuilder_BuiltChunk",
-		"MixinClientBossBar",
-		"MixinClientCommandSource",
-		"MixinClientPlayerEntity",
-		"MixinClientPlayerInteractionManager",
-		"MixinClientPlayNetworkHandler",
-		"MixinClientWorld",
-		"MixinClientWorld_Properties",
-		"MixinCloneCommand",
-		"MixinCommandBlockScreen",
-		"MixinCreativeInventoryScreen",
-		"MixinDimensionEffects_Nether",
-		"MixinEntity",
-		"MixinEntityRenderDispatcher",
-		"MixinExplosion",
-		"MixinFillCommand",
-		"MixinGameRenderer",
-		"MixinHeldItemRenderer",
-		"MixinInGameHud",
-		"MixinItem",
-		"MixinItemEntity",
-		"MixinItemGroup",
-		"MixinItemStack",
-		"MixinKeyboardInput",
-		"MixinLightingProvider",
-		"MixinLivingEntity",
-		"MixinMinecraftClient",
-		"MixinMobSpawnerBlockEntityRenderer",
-		"MixinMobSpawnerLogic",
-		"MixinMouse",
-		"MixinNetherPortalBlock",
-		"MixinObserverBlock",
-		"MixinParticleManager",
-		"MixinPlayerAbilities",
-		"MixinPlayerEntity",
-		"MixinRavagerEntity",
-		"MixinScaffoldingBlock",
-		"MixinSculkSensor",
-		"MixinServerPlayNetworkHandler",
-		"MixinShulkerBoxBlock",
-		"MixinSignBlockEntity",
-		"MixinSlimeBlock",
-		"MixinStructureBlockBlockEntity",
-		"MixinTeleportSpectatorMenu",
-		"MixinThreadedAnvilChunkStorage",
-		"MixinTradeOffer",
-		"MixinUpdateStructureBlockC2SPacket",
-		"MixinWindow",
-		"MixinWorld",
-		"MixinWorldRenderer"
-	],
-	"mixinPriority": 990,
-	"injectors": {
-		"defaultRequire": 0
-	}
+  "required": true,
```

Comment:
Since the `@Invoker` is not needed, you can revert this file entirely. In case it was needed, you should avoid making these whitespace changes.